docs/source/02_alfasim_sdk_by_example.rst

```diff
@@ -197,7 +197,18 @@ needed to load and unload the |sdk| API, in which will allows the plugin to use
 
     HOOK_FINALIZE(ctx)
     {
+
+        auto errcode = -1;
+        auto number_of_threads = -1;
+        errcode = alfasim.get_number_of_threads(ctx, &number_of_threads);
+        for (int thread_id = 0; thread_id < n_threads; ++thread_id) {
+            MyPluginModel* model = nullptr;
```

Comment:
I think that it'll be also good to add just above `HOOK_INITIALIZE` the following

```c++
struct MyPluginModel {
    double exponential = 0.0;
};
```

so that this become a complete example.

Reply:
Commit 1ace74af43fc17ca2f6161621660421d9cc2296d adds this struct ;)